[IMP] website: introduce `s_company_team_shapes` snippet and optimize team snippet images

Brieuc-brd opened the pull request:
### Optimize team snippet images 
This PR optimizes and adapts the images related to s_company_team_* snippet to avoid using multiple versions with differents sizes and ratios.

### Introduce s_company_team_shapes snippet 
This PR adds the new `s_company_team_shapes` snippet.

task-4140941
Part of task-4077427

Requires:
- https://github.com/odoo/design-themes/pull/888

---
![Capture d’écran 2024-08-29 à 15 20 58](https://github.com/user-attachments/assets/14af76c4-ef4e-47b4-8dff-991905200085)

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

anso-odoo commented on addons/website/views/snippets/s_company_team_shapes.xml:
```diff
@@ -0,0 +1,40 @@
+                </div>
+                <div class="col-6 col-lg-4 pb24">
+                    <img src="/web_editor/image_shape/website.s_company_team_image_1/web_editor/geometric_round/geo_round_square.svg" class="img img-fluid" alt="" data-shape="web_editor/geometric_round/geo_round_square" data-original-mimetype="image/jpeg" data-file-name="s_company_team_image_1.svg" data-shape-colors=";;;;" style="width: 100% !important;"/>
+                    <h3 class="h5-fs" style="text-align: center;"><br/>Tony Fred</h3>
```
You can wrap the image in a `<p>` to get it's margin do the spacing you want, without having to add a `<br/>` 👍 Same for other occurences

anso-odoo commented on addons/website/views/snippets/snippets.xml:
```diff
@@ -147,6 +147,7 @@
                 <t t-snippet="website.s_quotes_carousel" string="Quotes" group="people">
                     <keywords>testimonials</keywords>
                 </t>
+                <t t-snippet="website.s_company_team_shapes" string="Team Shapes" group="people"/>
```
I think we can add some keywords so it can be found more easily during the the search. I suggest:
`<keywords>organization, structure, people, team, name, role, position, image, portrait, photo, employees, shapes</keywords>`

anso-odoo reviewed (changes requested):
Hey 👋 Nice job, two small comments for you

Brieuc-brd commented:
Thanks @anso-odoo !
I've updated the branches, could you have a look ?
I also moved the snippet next to `company_team` in the modal list 👍 

The diffs :
- COM: https://github.com/odoo/odoo/compare/0e1b41a235bf5e69a94e6e4effb7d1269c968ef5..064283b035f7461ba0d0c889b868d7b856ee5031
- DT: https://github.com/odoo/design-themes/compare/f0843055069932148615061ccd2b4586a29389b1..a2e155dfb62ca3c8944cf6752cc242ec74124251

anso-odoo reviewed (approved):
Nice, LGTM 👍

sobo-odoo commented:
@robodoo delegate=bvr-odoo

bvr-odoo reviewed (approved):
@Brieuc-brd LGTM thanks :+1:

bvr-odoo commented:
@robodoo r+

robodoo commented:
@Brieuc-brd @bvr-odoo linked pull request(s) odoo/design-themes#888 not ready. Linked PRs are not staged until all of them are ready.

robodoo commented:
@Brieuc-brd @bvr-odoo because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

bvr-odoo commented:
@robodoo rebase-ff r+

robodoo commented:
Merge method set to rebase and fast-forward.

robodoo commented:
This PR is already reviewed, reviewing it again is useless.
